web3.js/test/transaction.test.ts

```diff
@@ -112,6 +114,28 @@ describe('Transaction', () => {
     });
   });
 
+  it('getEstimatedFee', async () => {
+    const connection = new Connection('https://api.testnet.solana.com');
+    const accountFrom = Keypair.generate();
+    const accountTo = Keypair.generate();
+
+    const {blockhash} = await helpers.latestBlockhash({connection});
+
+    const transaction = new Transaction({
+      feePayer: accountFrom.publicKey,
+      recentBlockhash: blockhash,
+    }).add(
+      SystemProgram.transfer({
+        fromPubkey: accountFrom.publicKey,
+        toPubkey: accountTo.publicKey,
+        lamports: 10,
+      }),
+    );
+
+    const fee = await transaction.getEstimatedFee(connection);
+    expect(fee).to.eq(5000);
```

Comment:
Since this test targets testnet, is this number bound to change sometime in the future and cause this test to start failing?

Reply:
When fees change, I expect this test to start failing. In that case, it may be hard to deterministically test this functionality